moninag/daemons/notifdaemon.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import time
+
+import django
+
+from daemon import Daemon
+
+
+# Path to project directory where manage.py is located
+project_path = '../'
+sys.path.append(project_path)
+
+# This is so Django knows where to find stuff
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'moninag.settings')
+django.setup()
+
+
+from service.models import Service
```

Comment:
I think there is no need to move it in separate module at this moment. In future, we could always do this if necessary.